rootfs/scheduler/__init__.py

```diff
@@ -585,19 +585,20 @@ def _set_container(self, namespace, container_name, data, **kwargs):
             data["resources"]["limits"]["cpu"] = cpu.lower()
 
         # add in healthchecks
-        healthchecks = kwargs.get('healthcheck', None)
-        if healthchecks and kwargs.get('routable', False):
-            # check if a port is present. if not, auto-populate it
-            # TODO: rip this out when we stop supporting deis config:set HEALTHCHECK_URL
-            if (
-                healthchecks.get('livenessProbe') is not None and
-                healthchecks['livenessProbe'].get('httpGet') is not None and
-                healthchecks['livenessProbe']['httpGet'].get('port') is None
-            ):
-                healthchecks['livenessProbe']['httpGet']['port'] = env['PORT']
-            data.update(healthchecks)
-        else:
-            self._default_readiness_probe(data, kwargs.get('build_type'), env.get('PORT', None))
+        if kwargs.get('routable', False):
```

Comment:
since the apps are not routable they shouldn't be having any readiness checks as it doesn't make any sense because readiness checks just makes sure that service endpoints are updated.
I will make another change with the health checks per proctype where an app can have a healthcheck even though it is not routable but we don't add a default healthcheck.